static/styles/zulip.css

```diff
@@ -1388,10 +1388,127 @@ blockquote p {
     font-weight: bold;
 }
 
+.alert-box {
```

Comment:
This is a pretty sizable, cohesive chunk of CSS that we're adding.  Should we make a new CSS file for this?